airbyte-cdk/python/unit_tests/sources/file_based/scenarios/csv_scenarios.py

```diff
@@ -34,6 +35,158 @@
+                                        "type": "string"
+                                    }
+                                },
+                                "validation_policy": {
+                                    "title": "Validation Policy",
+                                    "type": "string"
+                                },
+                                "catalog_schema": {
+                                    "title": "ConfiguredAirbyteCatalog",
+                                    "type": "object",
+                                    "properties": {
+                                        "streams": {
+                                            "title": "Streams",
+                                            "type": "array",
+                                            "items": {
+                                                "$ref": "#/definitions/ConfiguredAirbyteStream"
+                                            }
+                                        }
+                                    },
+                                    "required": ["streams"]
+                                },
+                                "input_schema": {
+                                    "title": "Input Schema",
+                                    "type": "object"
+                                },
+                                "primary_key": {
+                                    "title": "Primary Key",
+                                    "anyOf": [
+                                        {
+                                            "type": "string"
+                                        },
+                                        {
+                                            "type": "array",
+                                            "items": {
+                                                "type": "string"
+                                            }
+                                        },
+                                        {
+                                            "type": "array",
+                                            "items": {
+                                                "type": "array",
+                                                "items": {
+                                                    "type": "string"
+                                                }
+                                            }
+                                        }
+                                    ]
+                                },
+                                "max_history_size": {
+                                    "title": "Max History Size",
+                                    "type": "integer"
+                                },
+                                "days_to_sync_if_history_is_full": {
+                                    "title": "Days To Sync If History Is Full",
+                                    "type": "integer"
```

Comment:
nit: I think this should have a default value

Reply:
fair, i'll add the default value that the cursor uses if this is empty which is 3